basin3d/synthesis.py

```diff
@@ -659,7 +636,8 @@ def get_timeseries_data(synthesizer: DataSynthesizer, location_lat_long: bool =
             shutil.rmtree(working_directory)
 
 
-def _output_df(output_directory, output_name, query_info, metadata_store, first_timestamp, last_timestamp, temporal_resolution) -> \
+def _output_df(output_directory, output_name, query_info, metadata_store, first_timestamp, last_timestamp,
+               temporal_resolution) -> \
```

Comment:
minor: weird line break??

Reply:
Just formatting. I can remove but it is harmless.

Reply:
just jarring to read. I had to look twice to make sure there wasn't something amiss. As mentioned it is minor.

Reply:
k. Yeah. this is a function of the autoformatting

Reply:
same as below. it is geared to keep the line length reasonable. It is common practice but of course subject to opinion! :)